### ISSUE TITLE
unquoted value in url() function breaks highlighting

### ISSUE THREAD

**Issue by dwhieb** [atom/language-less#87]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Using unquoted values within the CSS `url()` function breaks syntax highlighting for LESS files from that point forward. The rest of the file is highlighted as a string. According to the [MDN docs on the `src` attribute](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src), the value of the `url()` function may be either a quoted or unquoted string, so the example below is valid CSS syntax.

### Steps to Reproduce

1. Start Atom and create a new LESS file.
2. Paste in the following code, which uses the CSS `url()` function.

```less
@font-face {
  font-family: 'Fira Sans';
  src:
    local('Fira Sans Light'),
    url(fonts/FiraSans-Light.woff2) format('woff2'),
    url(fonts/FiraSans-Light.woff) format('woff');
  font-display: swap;
  font-stretch: normal;
  font-style: normal;
  font-variant: normal;
  font-weight: 300;
}
```

**Expected behavior:** The properties after the `src` property should have syntax highlighting, as shown in the screenshot below.

![image](https://user-images.githubusercontent.com/5892997/35188723-e1f7e0f8-fe00-11e7-8da4-112f97d0a2d1.png)

**Actual behavior:** Everything after the `url()` function is treated as a string, and the rest of the document is given green highlighting, as shown in the screenshot below.

![image](https://user-images.githubusercontent.com/5892997/35188730-0c8e74c6-fe01-11e7-9271-7247bc407e62.png)

**Reproduces how often:** Reproduces 100% of times I have used the `url()` function with an unquoted value. It's possible this occurs with other CSS functions as well, but I haven't tested this yet.

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

**Atom version** 
Atom    : 1.19.0
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0

**apm version:** 1.18.2

*OS:* Windows 10 Pro v. 1709 build 16299.192

### Additional Information
This issue does not occur with files that have a .css extension or that Atom has been instructed to interpret as CSS. This issue appears to be unique to the `language-less` package.

I haven't contributed to Atom before but would be interested in working on this if it seems like a good beginner issue.

**Comment by 50Wliu** [comment]
I can't reproduce this:
![less-unquoted-url](https://user-images.githubusercontent.com/2766036/35189265-caa82552-fe14-11e7-91c2-b3ec10a8ee91.png)

In addition it looked like I fixed this in #76.  Since you're running an outdated version of Atom chances are if you update (to 1.23.3, currently the latest version) the issue should _magically_ disappear :).

**Comment by dwhieb** [comment]
Updating to the latest version of Atom did fix this. Sorry for the hassle - thought I was already on the latest version.
